test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -131,14 +91,14 @@ public class ContentResultTest
                         new MediaTypeHeaderValue("text/foo"),
                         "κόσμε",
                         null,
-                        "text/foo; charset=utf-8",
+                        "text/foo",
```

Comment:
Reasoning is a bit odd. User asked for _a_ `text/foo` content type and the framework chose to encode using UTF8 but didn't indicate that in the response headers.